.github/workflows/npmpublish.yml

```diff
@@ -0,0 +1,59 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Test on node 10 and ubuntu-latest
```

Comment:
I wonder if there is a way to re-use the  jobs/tasks from `nodejs.yml` to make maintaining easier.